fhd_core/setup_metadata/fhd_struct_init_obs.pro

```diff
@@ -187,7 +190,7 @@ struct={code_version:String(code_version),instrument:String(instrument),obsname:
     phasera:meta.phasera,phasedec:meta.phasedec,orig_phasera:meta.orig_phasera,orig_phasedec:meta.orig_phasedec,$
     n_pol:Fix(n_pol,type=2),n_tile:Long(n_tile),n_tile_flag:Long(n_flag),n_freq:Long(n_freq),n_freq_flag:0L,n_time:Long(n_time),n_time_flag:n_time_cut,$
     n_vis:Long(n_vis),n_vis_in:Long(n_vis_in),n_vis_raw:Long(n_vis_raw),nf_vis:Long(n_vis_arr),primary_beam_area:Ptrarr(4),primary_beam_sq_area:Ptrarr(4),pol_names:pol_names,$
-    jd0:meta.jd0,max_baseline:Float(max_baseline),min_baseline:Float(min_baseline),delays:meta.delays,lon:meta.lon,lat:meta.lat,alt:meta.alt,$
+    jd0:meta.jd0,max_baseline:max_baseline,min_baseline:min_baseline,delays:meta.delays,lon:meta.lon,lat:meta.lat,alt:meta.alt,$
```

Comment:
If min and max baseline are sometimes going to be doubles, then we need to force them to always be doubles or else the obs structure will sometimes be unreadable.